### PR TITLE
feat(1226): changed hardcoded contact us list to user-group api list

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Closes/Fixes/Resolves

### 

Closes ONEPLAT-1226

# Explain the feature/fix

* Converted hard-coded contact us list in one platform home-spa to user group api fetched list.
* List is sorted on alphabetical order of name.

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

Screenshot 2021-06-30 at 5 34 48 PM

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: packages/home-spa/src/service/service.js [hunk 1]
```diff
@@ -10,30 +10,46 @@ query HomePageQuery {
   }
 }`;
 
-export function getData() {
+export const getContactUsTeamDetails = `
+query ContactUsQuery{
+  group (cn:"one-portal-devel") {
+    cn
+    name
```
Comment (ghanlohar) [inline packages/home-spa/src/service/service.js:17]: The name is a deprecated property. you can use `cn` instead.

File: packages/home-spa/src/service/service.js [hunk 2]
```diff
@@ -10,30 +10,46 @@ query HomePageQuery {
...
+    members {
+      name
```
Comment (ghanlohar) [inline packages/home-spa/src/service/service.js:19]: Use `cn` here as the name is deprecated.

**Review by ghanlohar — CHANGES_REQUESTED** [review]

File: packages/home-spa/src/service/service.js [hunk 3]
```diff
@@ -10,30 +10,46 @@ query HomePageQuery {
...
     },
     body: JSON.stringify({
-      query: getHomeTypeBySPA
-    })
+      query,
+    }),
   };
-  return fetch(process.env.API_URL, fetchOptions)
+  return fetch( process.env.API_URL, fetchOptions )
     .then(res => res.json());
```
Comment (ghanlohar) [inline packages/home-spa/src/service/service.js:41]: I know you didn't add this but we should handle HTTP failures as fetch does not handle those more [here](https://www.tjvantoll.com/2015/09/13/fetch-and-errors/#:~:text=Per%20MDN%2C%20the%20fetch(),such%20a%20DNS%20lookup%20failure.)

**Review by ghanlohar — CHANGES_REQUESTED** [review]

File: packages/home-spa/src/service/service.js (hunk 3 above)
Comment (akhilmhdh) [reply packages/home-spa/src/service/service.js:41]: @ghanlohar yah thanks for sharing that ill make these changes with the other one 😄

Comment (akhilmhdh) [reply packages/home-spa/src/service/service.js]: Added an errorhandler part for both data fetch. For now if error occurs it gets consoled like other SPA.

**Comment by riginoommen** [comment]
Nikesh name should come first

**Review by riginoommen — APPROVED** [review]

**Review by hybridx — APPROVED** [review]

**Review by hybridx — APPROVED** [review]

**Review by ghanlohar — APPROVED** [review]

**Pull request merged**